geoviews/element/geo.py

```diff
@@ -8,6 +8,27 @@
 from holoviews.element import Text as HVText
 from iris.cube import Cube
 
+geographic_types = (cGoogleTiles, cFeature)
+
+def is_geographic(dataset, kdims=None):
+    """
+    Small utility that determines whether the supplied dataset
+    and kdims represent a geographic coordinate system.
+    """
+    if kdims:
+        kdims = [dataset.get_dimension(d) for d in kdims]
+    else:
+        kdims = dataset.kdims
+
+    if isinstance(dataset.data, geographic_types) or isinstance(dataset, WMTS):
+        return True
+    if (len(kdims) == 2 and
+        ((isinstance(dataset, _Element) and kdims == dataset.kdims) or
+        (isinstance(dataset.data, iris.cube.Cube) and all(dataset.data.coord(
```

Comment:
So one of the conditions is that if the dataset is an ``_Element``, it is geographic if the list of kdims matches the dataset kdims. Won't this latter condition always be true if the kdims argument isn't supplied (you have ``else: kdims = dataset.kdims`` above).